pkg/tablebackup/relinfo.go

```diff
@@ -1,8 +1,12 @@
 package tablebackup
 
 import (
+	"fmt"
 	"github.com/jackc/pgx"
```

Comment:
empty line before

Reply:
Fixed in the latest commit (not sure why GitHub still shows the old version).